src/js/pagination.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-use-before-define */
+import { getScores } from './data';
+
+let current = 1;
+
+export const display = async (data) => {
+  const tableBody = document.querySelector('.table-body');
+  const paginationUl = document.querySelector('.pagination');
+  const paginatedItems = paginate(current, 5, data);
+
+  tableBody.innerHTML = '';
+  paginatedItems.forEach((player) => {
+    const tr = document.createElement('tr');
+    tr.className = 'tr-row';
+    tr.innerHTML = `<td class="d-flex align-items-center"> <span class="user-icon"><i class="fa fa-user-circle"></i></span>${player.user} : ${player.score}</td>`;
+    tableBody.appendChild(tr);
+  });
+
+  const paginationSize = Math.ceil(data.length / 5);
+  pages(paginationSize, paginationUl);
+};
+
+export function pages(pages, paginationUl) {
+  paginationUl.innerHTML = '';
+  for (let i = 1; i <= pages; i += 1) {
+    const paginationLi = document.createElement('li');
+    paginationLi.className = 'p-item rounded-circle text-center d-flex justify-content-center';
+    paginationLi.id = i;
+    // eslint-disable-next-line eqeqeq
+    if (current == paginationLi.id) {
+      paginationLi.classList.add('bg-primary');
+      paginationLi.classList.add('p-active');
+    }
+    // eslint-disable-next-line no-loop-func
+    paginationLi.addEventListener('click', async () => {
+      current = paginationLi.id;
+      const result = await getScores();
+      const { result: data } = await result.json();
+      display(data);
+    });
+    paginationLi.innerHTML = `${i}`;
+    paginationUl.appendChild(paginationLi);
+  }
+}
+
+function paginate(currentPage = 1, rows, array) {
+  currentPage -= 1;
+  const loopStart = rows * currentPage;
+  const paginatedItems = array.slice(loopStart, loopStart + rows);
+  return paginatedItems;
+}
```

Comment:
- [ ] Kindly change these functions to es6 arrow functions, as required under the [JavaScript best practices guide](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.) to use es6 syntaxes.